### ISSUE TITLE
Schema names starting with Hex numbers generate invalid TypeScript types

### ISSUE THREAD

**Issue by ikeretsman** [RobinBlomberg/kysely-codegen#151]
Hi there 👋 !

I'm trying to use this library (version: 0.14.2) to generate types for [MUD-Indexer](https://mud.dev/services/indexer) database. Schemas there are created as addresses of world contract. And when I try to generate types I get this:

``` typescript
export interface 0x7203e7adfdf38519e1ff4f8da7dcdc969371f377Admin {
  __key_bytes: Buffer;
  __last_updated_block_number: Numeric | null;
  id: Buffer;
  value: boolean;
}
```

And obviously it cannot compile, because interface name can't start with number

I wonder if it's possible generate types for specific schema without explicit naming of this schema?

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/RobinBlomberg) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/RobinBlomberg/kysely-codegen/issues/151">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/151/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/151/pledge.svg">
</picture>
</a>

**Comment by RobinBlomberg** [comment]
This will be released in `kysely-codegen@0.15.0` soon.

**Comment by RobinBlomberg** [comment]
Should be released in [kysely-codegen@0.16.0](https://github.com/RobinBlomberg/kysely-codegen/releases/tag/0.16.0)! 🚀
